pulsar-client/src/main/java/org/apache/pulsar/client/impl/PersistentAcknowledgmentsGroupingTracker.java

```diff
@@ -267,10 +194,9 @@ private CompletableFuture<Void> doIndividualAck(MessageIdImpl messageId, Map<Str
     }
 
 
-    private CompletableFuture<Void> doIndividualAckAsync(MessageIdImpl messageId) {
+    private void doIndividualAckAsync(MessageIdImpl messageId) {
```

Comment:
How can an `Async` variant be `void`? Especially `doIndividualBatchAck` below returns `CompletableFuture<Void>`.

Reply:
The current names of these two "Async" methods are confusing. I will rename them.

Reply:
Since this PR already includes some refactorings, I might do that in a following PR.

Reply:
@BewareMyPower Yeah. That sounds good.